packages/amplify-console-integration-tests/src/profile-helper.ts

```diff
@@ -62,31 +63,33 @@ export function setupAWSProfile() {
   Object.keys(credentials).forEach(key => {
     const keyName = key.trim();
     if (profileName === keyName) {
-      credentials[key].aws_access_key_id = process.env.CONSOLE_AWS_ACCESS_KEY_ID;
-      credentials[key].aws_secret_access_key = process.env.CONSOLE_AWS_SECRET_ACCESS_KEY;
+      credentials[key].aws_access_key_id = process.env.AWS_ACCESS_KEY_ID;
+      credentials[key].aws_secret_access_key = process.env.AWS_SECRET_ACCESS_KEY;
+      credentials[key].aws_session_token = process.env.AWS_SESSION_TOKEN;
       isCredSet = true;
     }
   });
   if (!isCredSet) {
     credentials[profileName] = {
-      aws_access_key_id: process.env.CONSOLE_AWS_ACCESS_KEY_ID,
-      aws_secret_access_key: process.env.CONSOLE_AWS_SECRET_ACCESS_KEY,
+      aws_access_key_id: process.env.AWS_ACCESS_KEY_ID,
+      aws_secret_access_key: process.env.AWS_SECRET_ACCESS_KEY,
+      aws_session_token: process.env.AWS_SESSION_TOKEN,
     };
   }
 
-  process.env.CONSOLE_REGION = process.env.CONSOLE_REGION || testRegionPool[Math.floor(Math.random() * testRegionPool.length)];
+  process.env.CLI_REGION = process.env.CLI_REGION || testRegionPool[Math.floor(Math.random() * testRegionPool.length)];
```

Comment:
This environment variable is set separately as we have some custom allow list for our console e2e account to provision more Amplify App. This allow list is region specific and we run it in a region where there are not much traffic. If we want to reuse the `CLI_REGION`, then it should be overriden in the console e2e test

Reply:
Ah, we can have multiple if we want. I was just hoping to clear up any redundancies.